src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -71,7 +71,6 @@ internal static class ParameterHelpers
                 diagnostics,
                 allowRefOrOut: true,
                 allowThis: false,
-                // PROTOTYPE(func-ptr): Investigate if cases where this can be false for regular parameters applies to func pointers as well
```

Comment:
InAttribute cannot avoid being emitted for function pointer types, as it's part of the signature. Existing places that avoid it are non-virtual methods, which can also emit metadata on the parameter for this case. Function pointers have no other location to encode this, so we must do it here.

Reply:
> InAttribute cannot avoid being emitted for function pointer types, as it's part of the signature. Existing places that avoid it are non-virtual methods, which can also emit metadata on the parameter for this case. Function pointers have no other location to encode this, so we must do it here.

Consider capturing this in a comment, so that the next person wouldn't have to figure this out again

---
In reply to: [408483652](https://github.com/dotnet/roslyn/pull/43355#discussion_r408483652) [](ancestors = 408483652)